examples/getting_started_step_1/Cargo.toml

```diff
@@ -5,7 +5,6 @@ authors = ["Sean Griffin <sean@seantheprogrammer.com>"]
 
 [dependencies]
 diesel = "0.7.1"
-diesel_codegen = "0.7.2"
+diesel_codegen = { version = "0.7.2", features = ["postgres"] }
 diesel_codegen_old = { path = "../../diesel_codegen_old", features = ["postgres"] }
```

Comment:
wrong pr, but: you need specify this with a version to publish it to crates.io, right?